tests/generators/VegetaLoadGenerator.ts

```diff
@@ -0,0 +1,78 @@
+import * as fs from 'fs';
+
+import OperationGenerator from './OperationGenerator';
+import Did from '../../src/lib/Did';
+import { getProtocol } from '../../src/Protocol';
+import Cryptography from '../../src/lib/Cryptography';
+
+/**
+ * Class for generating files used for load testing using Vegeta.
+ */
+export default class VegetaLoadGenerator {
+
+  /**
+   * Creates a Create request followed by an Update request for each DID.
+   */
+  public static async generateCreateLoadFiles (uniqueDidCount: number, endpointUrl: string, absoluteFolderPath: string) {
```

Comment:
Everything looks great, Can you please add the params defintion?

Reply:
One more thing, how are planning to use this method?

Reply:
Parameter description added.

I am using this for load testing request throughput and operation processing throughput.

Reply:
This can be used for both local and cloud load testing.